### ISSUE TITLE
Pagerank I - Exercises 7-9

### ISSUE THREAD

**Issue by mvalsania** [mikeizbicki/cmc-csci145-math166#162]
It makes sense to me to talk about simple eigenvalues, but the notion of a simple eigenvector seems a bit confusing. Would anyone mind clarifying what we mean exactly by simple eigenvectors and why we are focusing on simple eigenvectors as opposed to simple eigenvalues? (Some additional thoughts on Image 2)

<img width="819" alt="image" src="https://github.com/user-attachments/assets/fe4fa6e9-3d09-46fa-b46c-05e3f67080ff">
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/9f3688c3-ca02-460e-9325-0c612fc90563">

**Comment by buffeinstein** [comment]
I recall overhearing this question from someone else over class. I think the question is just a bit poorly phrased? I believe that this question is trying to get at the fact that the largest eigenvector has an eigenvalue that is simple.

**Comment by mikeizbicki** [comment]
You're right that I didn't explicitly define a simple eigenvector.  The definition is: An eigenvector is simple if the corresponding eigenvalue is simple.  That should be enough to complete the problems.
